### DISCUSSION TITLE
Suggestion: Use Mouse-3 Instead of Mouse-1 to avoid accidental triggers

### DISCUSSION THREAD

**Discussion by nehrbash** [minad/jinx#124] [Ideas]
I understand this is completely a personal preference, but I've had to remap `jinx-overlay-map jinx-correct` to `<mouse-3>` (right-click) instead of `<mouse-1>` (left-click) because I kept accidentally clicking false positives when moving the cursor with the mouse. I am often too lazy to add them to my dictionary.

I thought I'd suggest this change as a potentially more sensible default binding as it's harder to accidentally trigger.

**Comment by minad** [comment]
Good idea. Iirc I had chosen mouse-1 such that the misspellings behave like links. The plan was to remove this at some point and replace it with a right click context menu as alternative, which would be even more convenient for mouse usage.

EDIT: Added in https://github.com/minad/jinx/commit/71706642cdc0031daef2c9df63cf73be8134d691.
